rust/noosphere-gateway/Cargo.toml

```diff
@@ -33,8 +36,8 @@ tower = { workspace = true }
 tower-http = { workspace = true, features = ["cors", "trace"] }
 async-trait = "~0.1"
 async-stream = { workspace = true }
-tracing = { workspace = true }
 wnfs-namefilter = { version = "0.1.21" }
+wnfs-common = "=0.1.23"
```

Comment:
Any reason for the version pinning?

Reply:
Yes, actually. I should have left a comment, but WNFS deprecated the `wnfs-namefilter` crate quietly and then broke compatibility in `wnfs-common`. Pinning to this version gives us breathing room to migrate.

Reply:
https://github.com/subconsciousnetwork/noosphere/issues/626